extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -6,6 +6,107 @@
  */
 
 /**
+ * Restores infobox dialog info when going back from media dialog
+ *
+ * @param {ve.ui.WindowManager} win
+ * @param {ve.dm.WikiaTransclusionModel} model - original tranclusion model of infobox dialog
+ * @param {string} imageName - image name from media dialog
+ * @param {string} paramName - Name of a field (e.g. image1)
+ */
+function restoreInfoboxDialog( win, model, imageName, paramName ) {
```

Comment:
VE conventions would be to encapsulate these methods (can be static methods) within the `ve.ui.WikiaParameterPage` class rather than having them in the global state especially with some generic method names like `openDialog` and `getImageName` below which could conflict with other JS.